### DISCUSSION TITLE
Are there any build-in keyboard shortcuts on tauri?

### DISCUSSION THREAD

**Discussion by percy507** [tauri-apps/tauri#3724] [Q&A] [answered]
Eg: 

```bash
# copy
Command+c

# paste
Command+v
```

**Comment by percy507** [comment, marked as answer]
After some research, I find the answer.

> There is no build-in shortcuts.

If you want to add some basic shortcuts(eg: copy, paste, cut), you need add a menu for your app.

> Related issue: [Copy/Paste and Other Keyboard Shortcuts Do Not Work on MacOS](https://github.com/tauri-apps/tauri/issues/2397)

```rs
let my_app_menu = Menu::new()
    .add_native_item(MenuItem::Copy)
    .add_native_item(MenuItem::Cut)
    .add_native_item(MenuItem::Paste);
```

**Reply by FabianLars** [reply]
Just want to make it absolutely clear that this is macOS specific behaviour.

**Reply by percy507** [reply]
Yes, get it. It seems electron default have these shortcuts. So I feel confused when first using tauri.
